### PR TITLE
Update setup relayer node guide and Chain Connector API page

### LINKED ISSUE #1944: Update Setup Relayer node guide
### Description

- After the recent update of the chain connector plugin, the [Setup Relayer node](https://lisk.com/documentation/run-blockchain/setup-relayer.html) guide requires an update.
- Update the request and response data for all the `chainConnector` plugin's endpoints.

#### Affected branch / URL
`development`

#### Page location
https://lisk.com/documentation/run-blockchain/setup-relayer.html

#### Additional Information
The issue is dependent on: https://github.com/LiskHQ/lisk-sdk/pull/9198

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Resolves #1944

### CODE REVIEW

File: docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc [hunk 1]
```diff
@@ -149,6 +154,145 @@ Sample configuration::
 =====
 IMPORTANT: A node operator must add either the value of `*receivingChainIPCPath*` or `*receivingChainWsURL*` in the chain connector's config.
 
+=== Configuring the `system`
+
+Before we discuss how to set up a node to act as a relayer node, we need to understand the concept of prefixes, how to retrieve them, and how to calculate `inclusionProofKeys`.
+
+==== What are Prefixes and how to get them?
```
Comment (ishantiw) [inline docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc:163]: maybe to know more about prefixes we can simply refer to LIP https://github.com/LiskHQ/lips/blob/main/proposals/lip-0040.md#module-store-prefix

File: docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc [hunk 2]
```diff
@@ -149,6 +154,145 @@ Sample configuration::
...
+
+We can get the `MODULE_PREFIX` and the `SUBSTORE_PREFIX` from the xref:{url_system_getmetadata}[system_getMetadata] endpoint.
+Each module has a set of `stores` and each store has a unique `key`.
+After invoking the aforementioned endpoint, search for the `"stores"` property of the `interoperability` module.
+
+Then, look for the `outbox` store of the `interoperability` module.
```
Comment (ishantiw) [inline docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc]: reference for outbox substore https://github.com/LiskHQ/lips/blob/main/proposals/lip-0045.md#outbox-root-substore

Comment (ishantiw) [inline docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc]:
```suggestion
Then, look for the `outbox` store of the `interoperability` module and note down the value of `key` property.
```

File: docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc [hunk 3]
```diff
@@ -149,6 +154,145 @@ Sample configuration::
...
+----
+
+.Properties required to create "inclusionProofKeys"
+[cols="~,~",options="header",stripes="hover"]
+|===
+|Prefix
+|Description
+
+|`MODULE_PREFIX`
+| The first 8 bytes of the `key` value is the `MODULE_PREFIX`.
```
Comment (ishantiw) [inline docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc]:
```suggestion
| The first 4 bytes of the `key` value is the `MODULE_PREFIX`.
```

File: docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc [hunk 4]
```diff
@@ -149,6 +154,145 @@ Sample configuration::
...
+For example, in the above-mentioned snippet, the `MODULE_PREFIX` is: `83ed0d25`.
+
+|`SUBSTORE_PREFIX`
+|The last 4 bytes of the `key` value is the `SUBSTORE_PREFIX`.
```
Comment (ishantiw) [inline docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc]:
```suggestion
|The last 2 bytes of the `key` value is the `SUBSTORE_PREFIX`.
```

File: docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc [hunk 5]
```diff
@@ -149,6 +154,145 @@ Sample configuration::
...
+.Calculating "inclusionProofKeys"
+[source,javascript]
+----
+
+// 1. Calculate the MODULE_PREFIX using the Interoperability module.
+MODULE_PREFIX = Buffer.from('83ed0d25', 'hex')
+
+// 2. Calculate the SUBSTORE_PREFIX using the Outbox Substore.
+SUBSTORE_PREFIX = Buffer.from('0000', 'hex') ()
+
```
Comment (ishantiw) [inline docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc:232]: You can mention that MODULE_PREFIX+SUBSTORE_PREFIX can be found from the `getMetadata` modules, store's "key" part

**Review by ishantiw — CHANGES_REQUESTED** [review]
Overall it looks nice, just a few comments on adding some more information and references.

File: docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc (hunk 2 above)
Comment (TalhaMaliktz) [reply docs/modules/ROOT/pages/run-blockchain/setup-relayer.adoc]:
```suggestion
Then, look for the `outbox` store of the `interoperability` module and note down the value of the `key` property.
```

**Review by Tschakki — APPROVED** [review]
LGTM 👍

**Review by ishantiw — APPROVED** [review]
Nicely done 👏

**Pull request merged**